config/flipt.schema.cue

```diff
@@ -94,13 +96,20 @@ package flipt
 		conn_max_lifetime?: int
 	}
 
+	#lower: ["debug", "error", "fatal", "info", "panic", "trace", "warn"]
+	#all: #lower + [ for x in #lower {strings.ToUpper(x)}]
```

Comment:
```suggestion
	_#lower: ["debug", "error", "fatal", "info", "panic", "trace", "warn"]
	_#all: _#lower + [ for x in _#lower {strings.ToUpper(x)}]
```
I forgot to mention that you can combine the visibility `_` with the def `#`.
This means that we dont expose `lower` and `all` as keys on the FliptSpec.

Currently, I get this when I eval `cue eval -E flipt.schema.cue`:
```cue
#FliptSpec: {
    #authentication: {
        #authentication_cleanup: {}
        #authentication_oidc_provider: {}
    }
    #cache: {}
    #cors: {}
    #db: {}
    #lower: ["debug", "error", "fatal", "info", "panic", "trace", "warn"]
    #all: ["debug", "error", "fatal", "info", "panic", "trace", "warn", "DEBUG", "ERROR", "FATAL", "INFO", "PANIC", "TRACE", "WARN"]
    #log: {
        #log_level: "debug" | "error" | "fatal" | "info" | "panic" | "trace" | "warn" | "DEBUG" | "ERROR" | "FATAL" | "INFO" | "PANIC" | "TRACE" | "WARN"
    }
    #meta: {}
    #server: {}
    #tracing: {}
    #ui: {}
}
```

This will make those private definitions dissapear where they're not wanted making this:
```cue
#FliptSpec: {
    #authentication: {
        #authentication_cleanup: {}
        #authentication_oidc_provider: {}
    }
    #cache: {}
    #cors: {}
    #db: {}
    #log: {
        #log_level: "debug" | "error" | "fatal" | "info" | "panic" | "trace" | "warn" | "DEBUG" | "ERROR" | "FATAL" | "INFO" | "PANIC" | "TRACE" | "WARN"
    }
    #meta: {}
    #server: {}
    #tracing: {}
    #ui: {}
}
```